helm_deploy/values-dev.yaml

```diff
@@ -22,7 +22,6 @@ env:
   OAUTH_ROOT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
   COMMUNITY_ENDPOINT_URL: https://community-api-secure.test.delius.probation.hmpps.dsd.io/secure
   ASSESSMENT_ENDPOINT_URL: https://offender-dev.aks-dev-1.studio-hosting.service.justice.gov.uk
-  FLAGS_ENABLEDELIUSTIERUPDATES: true
```

Comment:
Have we got any way of disabling the community-api writes in production (or elsewhere)?

Reply:
This project should always emit the event, It's up the clients to decide how to handle events they subscribe to I think. I have raised https://dsdmoj.atlassian.net/browse/MAS-374 to move the feature toggle to the delius-updates service .

Reply:
The main thing is that we don't accidentally enable writeback when we deploy this to production